### PR TITLE
More unit tests for `exporter.go`

### PR DESCRIPTION
**Pull request by tisnik**
# Description

More unit tests for `exporter.go`

## Type of change

- Unit tests (no changes in the code)

## Testing steps

Done on CI.

## Checklist
* [ ] `make before_commit` passes
* [ ] updated documentation wherever necessary
* [ ] added or modified tests if necessary
* [ ] updated schemas and validators in [insights-data-schemas](https://github.com/RedHatInsights/insights-data-schemas) in case of input/output change

### CODE REVIEW

File: exporter_test.go [hunk 1]
```diff
@@ -31,6 +31,15 @@ import (
 	main "github.com/RedHatInsights/insights-results-aggregator-exporter"
 )
 
+const (
+	expectedVersionMessage       = "Insights Results Aggregator Cleaner version 1.0"
+	expectedAuthorsMessage       = "Pavel Tisnovsky"
+	expectedCopyrightMessage     = "Red Hat Inc."
+	expectedConfigurationMesage1 = "Driver"
```
Comment (dpensi) [inline exporter_test.go:38]: expectedConfigurationMesage1 should be expectedConfigurationMessage1

**Review by dpensi — APPROVED** [review]

File: exporter_test.go [hunk 2]
```diff
@@ -31,6 +31,15 @@ import (
...
+	expectedConfigurationMesage2 = "Username"
+	expectedConfigurationMesage3 = "Host"
```
Comment (juandspy) [inline exporter_test.go:40]: 2 and 3 should be "mes**s**age" too.

**Review by juandspy — APPROVED** [review]

File: exporter_test.go (hunk 2 above)
Comment (tisnik) [reply exporter_test.go]: Nice catch, fixed

File: exporter_test.go (hunk 1 above)
Comment (tisnik) [reply exporter_test.go]: fixed

**Pull request merged**